browser/ui/brave_ads/ad_notification_popup_handler.h

```diff
@@ -27,7 +31,14 @@ class AdNotificationPopupHandler final {
   static void Show(Profile* profile, const AdNotification& ad_notification);
 
   // Close the notification popup view for the given |notification_id|.
-  static void Close(const std::string& notification_id);
+  // |by_user| is true if the notification popup was closed by the user,
+  // otherwise false.
+  static void Close(const std::string& notification_id, const bool by_user);
```

Comment:
nit: we don't need to use `const ` to primitive type(`bool`)

Reply:
fixed